src/graph.py

```diff
@@ -345,16 +353,19 @@ def output_forest_as_networkx(self, vertex_indices, edge_indices):
             networkx.Graph: a networkx graph object
         """
 
+        if len(vertex_indices) == 0: 
+            return nx.empty_graph(0), nx.empty_graph(0)
```

Comment:
Ideally we wouldn't special case this. What part of the below code breaks if `vertex_indices` / `edge_indices` is empty?

Reply:
You can't do df.loc[x] if x is empty. 

https://github.com/fraenkel-lab/OmicsIntegrator2/blob/091d1f63998e975f6789488ff7216a46bd167569/src/graph.py#L369

Reply:
Strangely, `self.nodes[vertex_indices.node_index.values]` produces this error if `vertex_indices` is an empty df: 

![image](https://user-images.githubusercontent.com/7219365/39256645-69820df2-487d-11e8-9cba-b3781e8afec8.png)

Reply:
Furthermore, `print(vertex_indices.node_index.values)` gives `[]`.

Reply:
Okay, so normally `vertex_indices.node_index.values` stores `node_index`s as `int`s, but if the merge that generates `vertex_indices` produces an empty df, `vertex_indices.node_index.values` is an empty numpy array with `dtype='object'`. 

This causes the error, since `self.nodes` can only take integers or boolean.

Reply:
d8d8ab4d637763f2e93e8c4e2d0cc06193ef3473